### PR TITLE
Sink connector

### PR DESCRIPTION
**Pull request by kyumarss**
Hi @jcustenborder 

I've checked in the basic  implementation for the Sink leg of the connector ( #12 ).

I will be adding more features (e.g. allow use of queue name instead of exchange/routingkey in the config file) and documentation in the coming days/weeks.

In meantime, your feedback is welcome.

### CODE REVIEW

**Comment by jcustenborder** [comment]
Thanks for the pull @kyumarss! Much appreciated. I added a couple comments.

**Comment by kyumarss** [comment]
Thanks for the quick reply.

For some strange reason, I don't see your comments. Any idea what the cause might be?

**Comment by jcustenborder** [comment]
Heh no clue. I have a problem with noticing activity on github in general.

**Comment by kyumarss** [comment]
I will ask the Github Support team for help.

**Comment by jcustenborder** [comment]
Maybe that's why. Sorry!

**Comment by kyumarss** [comment]
No problem :)
Thanks for the good comments. I'll soon push fixes.

File: src/main/java/com/github/jcustenborder/kafka/connect/rabbitmq/RabbitMQSinkTask.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    return VersionUtil.version(this.getClass());
+  }
+
+  @Override
+  public void put(Collection<SinkRecord> sinkRecords) {
+    for (SinkRecord record : sinkRecords) {
+      log.trace("current sinkRecord value: " + record.value());
+      //assumption: value is a string-convertible object (i.e., a valid JSON)
+      try {
+        channel.basicPublish(this.config.exchange, this.config.routingKey, null, record.value().toString().getBytes());
```
Comment (kyumarss) [inline src/main/java/com/github/jcustenborder/kafka/connect/rabbitmq/RabbitMQSinkTask.java]: I'm thinking that writing the transform might be cumbersome and non-trivial.
Wouldn't a converter/builder similar to the [DynamoDB connector](https://github.com/shikhar/kafka-connect-dynamodb/blob/master/src/main/java/dynamok/sink/AttributeValueConverter.java) connector or your own [Slor connector](https://github.com/jcustenborder/kafka-connect-solr/blob/master/src/main/java/com/github/jcustenborder/kafka/connect/solr/SolrInputDocumentBuilder.java) be preferred?

**Comment by jcustenborder** [comment]
Both of those have very defined structures for what you are writing to.
Take SolrDocument it looks very similar to a struct. For this connector to
work in all environment we would want to decouple the serialization from
the actual sending of the message. So for example someone could easily use
a transform that converts a struct to json text which would be handed to
the contents of the record passed to put. Transforms allow you to modify
the record in memory and leave it in it's original form on the topic. For
example you might have connect structs that are stores as avro in kafka,
but heading over to this system it needs to be json, or xml. A transform is
your ticket for that conversion.

**Comment by kyumarss** [comment]
I pushed fixes for the two issues mentioned in the review.

For testing the  connector, I couldn't see immediately how any of the standard SMTs can be leveraged. 
So, I wrote a basic SMT (named `ValueToBytes`) to test things locally. 
I can push the custom SMT code to its own repo on Github, if deemed useful.

**Comment by jcustenborder** [comment]
@kyumarss Sorry about the delay here. Are you ready for this to be merged?

**Comment by jcustenborder** [comment]
closes #14

**Comment by kyumarss** [comment]
@jcustenborder 
No problem. 
Go ahead please. I will clean up and push my connect SMT to a separate repo in the coming days.

**Comment by jcustenborder** [comment]
Thanks @kyumarss

**Pull request merged**